addon/-private/dynamic-attribute-bindings.js

```diff
@@ -8,11 +8,10 @@ export default Mixin.create({
 
     let newAttributeBindings = [];
     for (let key in this.attrs) {
-      if (this.NON_ATTRIBUTE_BOUND_PROPS.indexOf(key) === -1 && this.attributeBindings.indexOf(key) === -1) {
+      if (this.NON_ATTRIBUTE_BOUND_PROPS.indexOf(key) === -1 && this.attributeBindings.indexOf(key) === -1 && key !=='class') {
```

Comment:
My thoughts exactly and then add `NON_ATTRIBUTE_BOUND_PROPS` to `concatenatedProperties`